src/envoy/mixer/http_filter.cc

```diff
@@ -101,18 +104,46 @@ void CreateAuthIssuers(
 
 }  // namespace
 
+/**
+ * All http mixer filter stats. @see stats_macros.h
+ */
+// clang-format off
+#define ALL_HTTP_MIXER_FILTER_STATS(COUNTER, GAUGE)                           \
+  COUNTER(total_check_calls)                                                  \
+  COUNTER(total_remote_check_calls)                                           \
+  COUNTER(total_blocking_remote_check_calls)                                  \
+  COUNTER(total_quota_calls)                                                  \
+  COUNTER(total_remote_quota_calls)                                           \
+  COUNTER(total_blocking_remote_quota_calls)                                  \
+  COUNTER(total_report_calls)                                                 \
+  COUNTER(total_remote_report_calls)                                          \
+// clang-format on
+
+/**
+ * Struct definition for all mixer filter stats. @see stats_macros.h
+ */
+struct InstanceStats {
+  ALL_HTTP_MIXER_FILTER_STATS(GENERATE_COUNTER_STRUCT)
+};
```

Comment:
Maybe we can extract this into a separate header file so tcp_filter can use it too.  tcp has the same counters with a different prefix.